app/assets/javascripts/discourse/tests/unit/services/user-tips-test.js

```diff
@@ -1,15 +1,15 @@
 import { test } from "qunit";
 import pretender, { response } from "discourse/tests/helpers/create-pretender";
-import { getOwner } from "discourse-common/lib/get-owner";
+import { getOwnerWithFallback } from "discourse-common/lib/get-owner";
 import { acceptance } from "discourse/tests/helpers/qunit-helpers";
 
 acceptance("Unit | Service | user-tips", function (needs) {
```

Comment:
This should be a qunit `module()` instead and needs a `setupTest(hooks)` call - then the ownership will work